.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: python
 
-branches:
-    only:
-        - master
-
```

Comment:
I suppose it was set in an effort to avoid multiple travis jobs (which happens when sending PRs from the source repo).